src/frontend/src/common/analytics/eventTypes.ts

```diff
@@ -128,6 +128,8 @@ export type AnalyticsTreeCreationNextstrain = {
   phylo_run_workflow_id: number;
   // Type of tree being created
   tree_type: string;
+  // Location of samples used for tree creation
+  location_id?: number;
```

Comment:
Per Vincent's recommendation, updated this to explicitly pass null if there isn't a value.  Also added a comment to the analytics event page to indicate why this might be null.